overdrive.py

```diff
@@ -880,8 +954,75 @@ def book_info_to_metadata(cls, book, include_bibliographic=True, include_formats
             metadata.circulation = circulationdata
 
         return metadata
+   
 
+class OverdriveAdvantageAccount(object):
+    """Holder and parser for data associated with Overdrive Advantage.
+    """
+    
+    def __init__(self, parent_library_id, library_id, name, type):
+        """Constructor.
+        
+        :param parent_library_id: The library ID of the parent Overdrive 
+            account.
+        :param library_id: The library ID of the Overdrive Advantage account.
+        :param name: The name of the library whose Advantage account this is.
+        :param type: Should be the literal string 'Library Advantage Account'
```

Comment:
I don't know for sure that it's the only allowed value. I think I could take that out--it's not important right now.